code/src/components/Amount.js

```diff
@@ -0,0 +1,20 @@
+import React, {useState} from 'react'
```

Comment:
Here you add the {useState} but you are not adding any states here, only sending them through props from the Form.js. Therefore I would recommend not adding it in the import. I get a warning message in my console if I am not using something :)